.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -0,0 +1,170 @@
+                                f"""\
+File {file_path} contains code coverage disable statement."""
+                            )
+                            code_coverage_found = True
+        elif os.path.isfile(item):
+            # If it's a file, check it directly
+            if (
+                item.endswith(".ts")
+                and not item.endswith(".test.ts")
+                and not item.endswith(".spec.ts")
+            ):
+                if has_code_coverage_disable(item):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)